app/controllers/order_detail_stored_files_controller.rb

```diff
@@ -48,19 +48,23 @@ def upload_order_file
         redirect_to(order_path(@order))
       end
     else
-      flash.now[:error] = "An error was encountered while uploading the Order File" # TODO: I18n
+      flash.now[:error] = text('upload_order_file.error')
       render :order_file
     end
   end
 
   # GET /orders/:order_id/order_details/:order_detail_id/remove_order_file
   def remove_order_file
-    if @order_detail.stored_files.template_result.all?(&:destroy)
-      flash[:notice] = "The uploaded Order File has been deleted successfully" # TODO: I18n
+    if @order_detail.new?
+      if @order_detail.stored_files.template_result.all?(&:destroy)
+        flash[:notice] = text('remove_order_file.success')
```

Comment:
Okay, I am genuinely mystified by this. I can't find anything in the rubocop config that explains why it should not have flagged on this.